otel-agent/k8s-helm/Chart.yaml

```diff
@@ -8,7 +8,7 @@ keywords:
   - Coralogix
 dependencies:
   - name: opentelemetry-collector
-    version: "0.52.0"
+    version: "0.55.0"
```

Comment:
It looks like the Domain enhancement was released only in `0.76.3` collector (https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.76.3).

Should we first bump the version in the Helm chart, since it's `0.76.1` right now?

Reply:
@povilasv has directed me to this comment https://github.com/open-telemetry/opentelemetry-helm-charts/pull/772#issuecomment-1531564324 explaining the versioning situation, looks like `0.76.1` is the correct version to use. Resolving this thread 👍.